README.md

```diff
@@ -774,6 +774,29 @@ Translations of the guide are available in the following languages:
   end
   ```
 
+* <a name="avoid-repeating-aliased-module-names"></a>
+  If you plan to alias your modules, avoid using the same name consecutively.
+  This avoids any problems with [Conflicting Aliases]
+  for the module.
+  <sup>[[link](#avoid-repeating-aliased-module-names)]</sup>
+
+  ```elixir
+  # not preferred
+  defmodule Todo.Todo do
+    alias Todo.Todo
+  end
+
+  # somewhat preferred
+  defmodule Todo.Models.Todo do
+    alias Todo.Models.Todo
+  end
+
+  # preferred
+  defmodule Todo.List do
+    alias Todo.List
+  end
+  ```
+
```

Comment:
I don't think this is clearly explained.
The whole reason for not repeating fragments in the module name is because when they are aliased, it could be ambiguous to which module you are refereing to.

examples.
```elixir
# not preferred
defmodule Todo.Todo do
  ...
end

defmodule AnotherModule do
  alias Todo.Todo
  # here it is ambigious to which one we refer when we call `Todo`
end

I think those would be more real life examples, as in the examples given the proper way to refer to module from within is using `__MODULE__` not matter what is called. the problem is when aliasing from another module.